submissions/xT0PERx/Memory pair game/transitions.js

```diff
@@ -0,0 +1,116 @@
+const imageses = [
+  'images/Alfa-romeo.jpeg',
+  'images/Ford.jpeg',
+  'images/Mazda.jpeg',
+  'images/Nissan.jpeg',
+  'images/Peugeot.jpeg',
+];
+//my random array with my img
+const cardDeck = imageses.concat(imageses);
+cardDeck.sort(function () {
+  return 0.5 - Math.random();
+});
+
+//how many pairs of images matched
+let winCount = 0;
+
+//this func must added imgs in all my cards
+const allCardsOnTheDesk = document.querySelector('main');
+const putImgsInCards = function () {
+  const fragment = document
+    .createDocumentFragment()
+    .appendChild(allCardsOnTheDesk);
+
+  for (let i = 0; i < cardDeck.length; i++) {
+    const img = document.createElement('img');
+    img.src = cardDeck[i];
+    fragment.children[i].appendChild(img);
+  }
+  document.body.appendChild(fragment);
+};
+
+// this function flips the cards
+
+const overturnsCard = function () {
+  allCardsOnTheDesk.addEventListener('click', function (event) {
```

Comment:
In my opinion, it was better like this 
```
const overturnsCard = ({target}) => {
......
}
allCardsOnTheDesk.addEventListener('click', overturnsCard)

```

`{target}` it is [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), you can read more about this

Reply:
I understood the idea itself, but I could not implement it. I read the article that you offered me, but unfortunately I couldn't fix the code according to your instructions. For some reason, the code didn't work. I committed it and left it in the file. Perhaps you can explain to me where I made a mistake.